tests/test_bootstrap.py

```diff
@@ -1196,7 +1196,7 @@ def test_BootstrapRedlConsistency_normalization(self):
         )
 
         # Results are not perfectly identical because ln(Lambda) is not quite invariant.
-        np.testing.assert_allclose(results, expected, rtol=2e-3)
+        np.testing.assert_allclose(results, expected, rtol=3e-3)
```

Comment:
Is this an acceptable loss of accuracy when reducing the number of L points?

Reply:
I think yes, we halve the nodes and only lose one part in 1000 relative accuracy in this test